tools/gen-release-notes

```diff
@@ -14,7 +14,7 @@ import subprocess
 # Ensure certain order for known sections.
 SECTIONS_TOP = ['core', 'memtx', 'vinyl', 'replication', 'swim',
                 'raft', 'luajit', 'lua', 'sql']
-SECTIONS_BOTTOM = ['build', 'testing', 'misc']
+SECTIONS_BOTTOM = ['build', 'testing', 'tools', 'misc']
```

Comment:
Nit: It is for direct subsections of `feature` and `bugfix`. Since we're going to move `testing` and `tools` into top level sections, I would rather remove `testing` from this list.

NB: Shouldn't we push the script changes via `master`?

Reply:
https://github.com/tarantool/tarantool/commit/4d6937ef69a55b72ce1c8194cf9c42b913752a73